test/controllers/benefits/CompanyBenefitControllerSpec.scala

```diff
@@ -194,6 +208,62 @@ class CompanyBenefitControllerSpec
       }
     }
 
+    "redirect to the Tax Account Summary Page (start of journey)" when {
+      "the form has the value noIDontGetThisBenefit and EndCompanyBenefit_BenefitTypeKey is not cached" in {
+        val SUT = createSUT
+        when(
+          journeyCacheService.mandatoryJourneyValueAs(Matchers.eq(EndCompanyBenefit_BenefitTypeKey), any())(
+            any()))
+          .thenReturn(Future.successful(Left("")))
+
+        val result = SUT.submitDecision(
+          RequestBuilder
+            .buildFakeRequestWithAuth("POST")
+            .withFormUrlEncodedBody(DecisionChoice -> NoIDontGetThisBenefit))
+
+        val redirectUrl = redirectLocation(result)
+
+        redirectUrl mustBe Some(controllers.routes.TaxAccountSummaryController.onPageLoad().url)
+
+      }
+      "the form has the value YesIGetThisBenefit and EndCompanyBenefit_BenefitTypeKey is not cached" in {
+        val SUT = createSUT
+        when(
+          journeyCacheService.mandatoryJourneyValueAs(Matchers.eq(EndCompanyBenefit_BenefitTypeKey), any())(
+            any()))
+          .thenReturn(Future.successful(Left("")))
+
+        val result = SUT.submitDecision(
+          RequestBuilder
+            .buildFakeRequestWithAuth("POST")
+            .withFormUrlEncodedBody(DecisionChoice -> YesIGetThisBenefit))
+
+        val redirectUrl = redirectLocation(result).getOrElse("")
+
+        redirectUrl mustBe controllers.routes.TaxAccountSummaryController.onPageLoad()
+
+      }
+      "the form has no valid value" in{
+        val SUT = createSUT
+        val benefitType = "Telephone"
+        when(
+          journeyCacheService.mandatoryJourneyValueAs[String](Matchers.eq(EndCompanyBenefit_BenefitTypeKey), any())(
```

Comment:
You can replace the `Matchers.eq` with named imports to make it a little more concise.
If you import `import org.mockito.Matchers.{eq=>eqTo}` then usage would be 
```suggestion
          journeyCacheService.mandatoryJourneyValueAs[String](eqTo(EndCompanyBenefit_BenefitTypeKey), any())(
```
As this is used a lot in this file it would make it much easier to read the code

Reply:
Added and replaced all instances in file